scripts/contract-upgrade-governance.sh

```diff
@@ -104,6 +104,12 @@ case "$chain_name" in
   solana)
     chain=1
     explorer="https://explorer.solana.com/address/"
+    extra=""
+    ;;
+  pythnet)
+    chain=26
+    explorer="https://explorer.solana.com/address/"
+    extra="Be sure to set the custom RPC to https://pythnet.rpcpool.com"
```

Comment:
custom RPC where? should we include more info here, or is it obvious to guardians?

Reply:
I added a few more words.